extensions/libraries/redcore/translation/table.php

```diff
@@ -1206,4 +1206,25 @@ public static function setInstalledTranslationTables($option, $table, $contentEl
 			$translationTableModel->save($data);
 		}
 	}
+
+	/**
+	 * Method to purge contents of a translation table
+	 *
+	 * @param   string  $tableName  name of either the translations table or the table being translated I.E. #__content or #__content__rctranslations
+	 *
+	 * @return bool
+	 */
+	public static function purgeContentElement($tableName)
```

Comment:
I didn't dig that deep. Just implemented the missing method used by redshopb. Should I just use that method instead?